hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/MetricsHBaseServerWrapperImpl.java

```diff
@@ -68,7 +68,7 @@ public int getNumOpenConnections() {
     if (!isServerStarted() || this.server.connectionList == null) {
```

Comment:
We can also remove `this.server.connectionList == null` because it is not relevant anymore (also, it can never be null)